std/typecons.d

```diff
@@ -2400,20 +2400,31 @@ struct Rebindable(T)
 if (!is(T == class) && !is(T == interface) && !isDynamicArray!T && !isAssociativeArray!T)
 {
 private:
-    align(T.alignof)
-    static struct Payload
+    static if (isAssignable!(typeof(cast() T.init)))
     {
-        static if (hasIndirections!T)
-        {
-            void[T.sizeof] data;
-        }
-        else
+        enum useReinterpretCast = true;
```

Comment:
```suggestion
        enum useQualifierCast = true;
```

https://dlang.org/spec/expression.html#cast_qualifier

Reply:
Btw, it's actually the `Payload` case below the one that is actually doing a reinterpret cast.

Reply:
er, yeah